### DISCUSSION TITLE
Call for new developers willing to actively contribute on Searx

### DISCUSSION THREAD

**Discussion by unixfox** [searx/searx#2474] [Ideas]
Currently, Searx has only one active developer which is doing all the work of maintaining the code of Searx, it's @dalf. We also have other contributors (@return42, @MarcAbonce and @kvch) that help us from time to time.

There are currently 22 open bugs, 159 feature requests opened and 29 pull requests to review. @dalf humanly can't solve all of them, we really need help from other developers.

I've been thinking of posting on multiple sub Reddit (/r/searx, /r/opensource, /r/degoogle/, /r/privacy/ and more) for asking for help.
What kind of other platform would be interesting to post on? We welcome any other ideas for bringing more developers on the project!

If you are also a developer familiar in Python and would like to be a maintainer of Searx please don't hesitate to write us a message here by responding to this post.

**Comment by dalf** [comment]
Here, I have to say that my work on searx wouldn't be possible without the sponsoring of the [NLnet Foundation](https://nlnet.nl/). 
A big thanks to them !

A list of milestones have been defined:
* https://github.com/searx/searx/wiki/Milestones (the milestones 1.1, 1.6 are done)
* https://github.com/searx/searx-stats2/wiki/Milestones
* And there is also #1703

I will try my best but one day has only 24 hours, so as @unixfox said, a new developer is welcomed.

**Comment by virtadpt** [comment]
Make a couple of posts to the Fediverse.  That'll get attention.

You might also want to post to the HOPE 2020 Matrix channels.
